app/scripts/modules/kubernetes/v2/manifest/wizard/basicSettings.component.ts

```diff
@@ -0,0 +1,56 @@
+import { IComponentOptions, IController, module } from 'angular';
+
+import { IKubernetesManifestCommand } from '../manifestCommandBuilder.service';
+
+class KubernetesManifestBasicSettingsCtrl implements IController {
+  public command: IKubernetesManifestCommand;
+}
+
+class KubernetesManifestBasicSettingsComponent implements IComponentOptions {
+  public bindings: any = {command: '='};
+  public controller: any = KubernetesManifestBasicSettingsCtrl;
+  public controllerAs = 'ctrl';
+  public template = `
+    <div class="container-fluid form-horizontal">
+      <ng-form name="basicSettings">
+        <div class="form-group">
+          <div class="col-md-3 sm-label-right">
+            Account
+          </div>
+          <div class="col-md-7">
+            <account-select-field component="ctrl.command"
+                                  field="account"
+                                  accounts="ctrl.command.backingData.accounts"
+                                  provider="'kubernetes'"></account-select-field>
+          </div>
+        </div>
+
+        <div class="form-group">
+          <div class="col-md-3 sm-label-right">
+            Application
+          </div>
+          <div class="col-md-7"><input readonly="true"
+                                       type="text"
+                                       class="form-control input-sm no-spel"
+                                       name="cluster"
+                                       ng-model="ctrl.command.moniker.app"/></div>
+        </div>
+
+        <div class="form-group">
+          <div class="col-md-3 sm-label-right">
+            Cluster
+          </div>
+          <div class="col-md-7"><input type="text"
+                                       class="form-control input-sm no-spel"
```

Comment:
Do you want `no-spel` here? (It prevents the expression helper from coming up).

Reply:
Good point